solc/CommandLineInterface.cpp

```diff
@@ -486,6 +486,12 @@ void CommandLineInterface::createFile(string const& _fileName, string const& _da
 		BOOST_THROW_EXCEPTION(FileError() << errinfo_comment("Could not write to file: " + pathName));
 }
 
+void CommandLineInterface::createJson(string const& _fileName, Json::Value const& _json)
```

Comment:
I think this would be generic having a flag to switch between `jsonCompactPrint` and `jsonPrettyPrint` given some outputs are pretty printed.